dags/nct.py

```diff
@@ -16,10 +19,40 @@
     schedule_interval='@monthly'
 )
 
+HTTP_CONN_ID = 'datastore_http'
+S3_BASE_URL = airflow.hooks.BaseHook.get_connection(HTTP_CONN_ID).host
+S3_URL_ENDPOINT = '/dumps/sources/nct_2001-01-01_{{ end_date }}.zip'
+NCT_DATA_URL = '{base}{endpoint}'.format(
+    base=S3_BASE_URL,
+    endpoint=S3_URL_ENDPOINT
+)
+
+save_nct_xml_to_s3_task = HTTPToS3Transfer(
+    task_id='save_nct_xml_to_s3',
+    dag=dag,
+    url='https://clinicaltrials.gov/search',
+    url_params={
+        'resultsxml': 'True',
+        'rcv_s': '01/01/2001',
```

Comment:
Food for thought: would be nice to have a way to configure these things easier. Some day.